### PR TITLE
Job scheduler changes

### PR DESCRIPTION
**Pull request by dungnmaster**
This PR adds support for creating and dropping jobs in evadb based on this: https://github.com/georgia-tech-db/evadb/issues/1248

### CODE REVIEW

**Comment by xzdandy** [comment]
Hi @dungnmaster, thanks for looking into Job scheduler. What is the status of the PR? Could you summarize the design which helps us understanding the implementation. Thanks!

**Comment by dungnmaster** [comment]
> Hi @dungnmaster, thanks for looking into Job scheduler. What is the status of the PR? Could you summarize the design which helps us understanding the implementation. Thanks!

- In the current implementation, the job scheduler process can be started using the start_jobs() method of the EvaDbConnection object and stopped using the stop_jobs() method.
- The start_jobs() method, starts a new Python process that keeps polling for valid executable tasks in the catalog and also executes them in the same process. This will continue until the job scheduler is stopped or the calling python parent process terminates.

File: evadb/utils/job_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+                    # update the next trigger time for this job
+                    self._update_next_schedule_run(next_executable_job)
+
+                    # update the previosly inserted job history record with endtime
+                    self._evadb.catalog().update_job_history_end_time(
+                        next_executable_job.row_id,
+                        execution_time,
+                        datetime.datetime.now(),
+                    )
```
Comment (xzdandy) [inline evadb/utils/job_scheduler.py:99]: We can update the result of job in the catalog. so it is queryable. We can do this in later PRs. Maybe add a TODO.

**Review by xzdandy — APPROVED** [review]
Great work! Very good quality. Could you also update the documentation https://evadb.readthedocs.io/en/stable/source/reference/evaql.html for the `CREATE JOB` query? Thanks!
